lib/domain/model/category/area_type.dart

```diff
@@ -13,23 +13,24 @@ class AreaType extends Category {
   Map<String, dynamic> toJson() => _$AreaTypeToJson(this);
 
   @override
-  String get name => switch (areaCode)  {
-    '99' => '내주변',
-    '1' => '서울',
-    '2' => '인천',
-    '3' => '대전',
-    '4' => '대구',
-    '5' => '광주',
-    '6' => '부산',
-    '7' => '울산',
-    '8' => '세종',
-    '32' => '강원도',
-    '33' || '34' => '충청도',
-    '35' || '36' => '경상도',
-    '37' || '38' => '전라도',
-    '39' => '제주도',
-    _ => '전체'
-  };
+  String get name => switch (areaCode) {
+        '99' => 'AI추천',
+        '1' => '서울',
+        '2' => '인천',
+        '3' => '대전',
+        '4' => '대구',
+        '5' => '광주',
+        '6' => '부산',
+        '7' => '울산',
+        '8' => '세종',
+        '9' => '경기',
```

Comment:
```suggestion
        '9' => '경기도',
```
피그마대로면 경기도가 맞는 것 같습니다

Reply:
오! 역시 👍👍👍
감사합니다